### PR TITLE
tests: remove opensuse 15 4

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Opensuse leap 15.4 is EoL (15.6 is comming soon)

### CODE REVIEW

**Review by andrewphelpsj — APPROVED** [review]
Thanks!

File: tests/lib/external/snapd-testing-tools/spread.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ backends:
                 storage: preserve-size
             - centos-9-64:
                 storage: preserve-size
-            - opensuse-15.4-64:
+            - opensuse-15.5-64:
```
Comment (olivercalder) [inline tests/lib/external/snapd-testing-tools/spread.yaml:38]: Should this be deleted outright, then 15.6 added when the time comes? I doubt we want a duplicate 15.5, unless this is intentionally a placeholder.

**Review by olivercalder — CHANGES_REQUESTED** [review]
Looks good, I think just one problem with duplicate 15.5. Thanks!

File: tests/lib/external/snapd-testing-tools/spread.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/lib/external/snapd-testing-tools/spread.yaml:38]: thanks @olivercalder , seems to be a problem with the merge. I'll fix the change

**Review by olivercalder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**